src/agent/src/watcher.rs

```diff
@@ -1291,7 +1291,7 @@ mod tests {
 
     #[tokio::test]
     #[serial]
-    #[cfg(not(target_arch = "aarch64"))]
+    #[cfg(not(any(target_arch = "aarch64", target_arch = "s390x")))]
```

Comment:
Are the tests failing for the same reasons ? We have #4248 for s390x but I couldn't find anything for aarch64 (Cc @jongwu ). It would be good to have the details in a comment to avoid the need for `git blame`.

Reply:
Sorry. A root cause has not yet been identified for `s390x`. I have no idea regarding how the failure was triggered for `aarch64`.

Reply:
Based on https://github.com/kata-containers/kata-containers/pull/6056#issuecomment-1382613020, it seems there is a dependency issue while running the tests concurrently.